frontend/src/helpers/tree_view/construct_tree.ts

```diff
@@ -42,6 +42,47 @@ export const build_path = (array: Array<any>, parent_id, path) => {
 }
 
 export const construct_tree = (node_list: Array<Node>): Array<any> => {
+<<<<<<< HEAD
```

Comment:
@vitalii-bulyzhyn this has a head tag  / document has has source control artifacts

Reply:
Just removed